<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Основное окно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidgetUsers">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>551</width>
      <height>321</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddUser">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить пользователя</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Профиль</string>
    </property>
    <addaction name="actionChangePassword"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChangePassword">
   <property name="text">
    <string>Изменить пароль</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
